src/View/Components/Button/Circle.php

```diff
@@ -21,17 +27,37 @@ public function __construct(
         public ?string $loading = null,
         public ?string $delay = null,
         public ?string $style = null,
+        public ?string $size = 'lg',
     ) {
         $this->style = $this->outline ? 'outline' : 'solid';
 
+        $this->buttonSizeClasses = [
```

Comment:
All this logic should be moved to `src/Views/Personalization/Support/Collors/ButtonsColors.php`